seed/data/notificationType/notificationTypes.js

```diff
@@ -0,0 +1,14 @@
+module.exports = [
```

Comment:
We need to remove these and set up the actual notification types that we currently support - one for each of the notification templates that we currently have. That is...

- Activity Report
- Transcript
- Request a Human (V1)
- Request a Human (V2)
- Follow Up Email